resources/js/layouts/CardLayout.vue

```diff
@@ -2,15 +2,21 @@
   <div class="flex min-h-full flex-col py-8 sm:px-6 lg:px-8 lg:py-16">
     <div class="sm:mx-auto sm:w-full sm:max-w-md">
       <img
-        class="mx-auto h-48 w-auto"
+        class="mx-auto h-48 w-auto dark:hidden"
         src="/images/logo.png"
         alt="FSR 5 Logo"
         loading="lazy"
       />
+      <img
+        class="mx-auto hidden h-48 w-auto dark:block"
+        src="/images/logo-dark.png"
+        alt="FSR 5 Logo"
+        loading="lazy"
+      />
       <h2
         class="mt-6 text-center text-3xl font-bold tracking-tight text-gray-900 dark:text-gray-100"
       >
-        Herzlich Willkommen in der Erstiwoche 2023
+        Herzlich Willkommen in der {{ title }} {{ new Date().getFullYear() }}
```

Comment:
If we set title to "Gerolstein" the sentence would be "Herzlich Willkommen in der Gerolstein 2023". I think the article before the title must be a variable or the sentence must be restructured.

Reply:
Fixed in new commit